### ISSUE TITLE
Instrumentation needs for Sonification/GT collaboration

### ISSUE THREAD

**Issue by ariel-phet** [phetsims/build-an-atom#140]
Related to #107 

@jbphet said:

> Sam, Emily, Jesse and I met with Brianna T from Georgia Tech today to discuss her needs for the BAA instrumentation.  Here are a couple of things that you may want to know from this meeting.
> - She figures they will need a version that supplies basic parameters for sonification in the Feb 2017 time frame.  The parameters she requested are, for the most part, accessible in the simulation and shouldn’t be hard to instrument.  In fact, many of them may already be instrumented.
> 
> - The project where they control the simulation via alternate user interface devices (Leap?) will be later in the year.  She didn’t have a clear sense of when, but I got the sense that it could be several months after that.
> 
> -  They are not currently planning to sonify the game.

**Comment by jbphet** [comment]
Brianna T also said:

>The variables we've been talking about for sonifications in Build an Atom are:
> 1. number of protons/neutrons/electrons
> 2. net charge (+/- ion or neutral)
> 3. stable/unstable
> 4. mass number

**Comment by jbphet** [comment]
Issue #107 is also about adding instrumentation, and is a more general version of this issue.

**Comment by jbphet** [comment]
I've published a version that I believe has the instrumentation necessary to support the items requested in comment https://github.com/phetsims/build-an-atom/issues/140#issuecomment-265225202.  That version can be found here: http://www.colorado.edu/physics/phet/dev/html/build-an-atom/1.3.0-phetiodev.1/

Assigning to @ariel-phet to determine whether we should test this before handing it over to Georgia Tech.  It seems to me like some amount of testing would probably be good.

**Comment by ariel-phet** [comment]
Yes, some testing would be good, please make a task, provide some context for QA and assign to me initially.

**Comment by jbphet** [comment]
Task created, see https://github.com/phetsims/tasks/issues/748.

**Comment by jbphet** [comment]
The testing has been completed.  After talking with @samreid, I learned that I really should have done an RC and production version for sharing with GT (this is apparently our policy for sharing phet-io versions), so I went ahead and created these versions and had them sanity tested.  The link has been provided to @emily-phet, and I will assign this to her so that she can either log any problems that arise once the GT folks start using it or close the issue if it meets their needs.

**Comment by jbphet** [comment]
By the way, I'm not going to put the link to the phet-io version in this GitHub issue since the issue is visible to the general public and the link is intended to be hidden.

**Comment by jbphet** [comment]
In a meeting with Brianna and others today, she expressed a need to be able to monitor the open/closed state of the accordion boxes.  This capability has been implemented in the 1.4.0-phetio release, but that release is not yet fully tested and is therefore not yet a production release.  I've let Brianna know about this and will provide her the production release when it's ready.

**Comment by emily-phet** [comment]
@jbphet can this be closed?

**Comment by jbphet** [comment]
Sure.  Closing.
